yarn.lock

```diff
@@ -0,0 +1,1663 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I'm very much not a fan of `yarn`, having had it break too many projects. Please remove this file from the PR, thanks!